_data/meltano/utilities/postgres/postgres.yml

```
@@ -0,0 +1,41 @@
+name: postgres
+namespace: postgres
+domain_url: https://www.postgresql.org/
+label: Postgres
+description: The World's Most Advanced Open Source Relational Database.
+docs: https://www.postgresql.org/
```

```suggestion
docs: https://www.postgresql.org/
definition: is a containerized instance of Postgres useful for local development.
```

Makes it so that the Hub renders something more that `the postgres utility`